cinema/urls.py

```diff
@@ -1 +1,22 @@
-# write urls here
+from django.urls import include, path
+from rest_framework import routers
+from .views import (
+    GenreViewSet,
+    ActorViewSet,
+    CinemaHallViewSet,
+    MovieViewSet,
+    MovieSessionViewSet,
+)
+
+router = routers.DefaultRouter()
+router.register("genres", GenreViewSet)
+router.register("actors", ActorViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+
+urlpatterns = [
+    path("", include(router.urls))
+]
```

Comment:
No need to use include if base path is not provided. Use:
`urlpatterns = router.urls`